server/routes/reviews.js

```diff
@@ -0,0 +1,77 @@
+const router = require("express").Router();
+const { User, Product, Review } = require("../db/index.js");
+
+//All reviews
+router.get("/", async (req, res, next) => {
+  try {
+    res.send(await Review.findAll());
+  } catch (err) {
+    next(err);
+  }
+});
+
+//Single review
+router.get("/:id", async (req, res, next) => {
```

Comment:
can we qualify what id means?
I feel that I can guess that this req.params.id is the productId. I'm just not sure.
So maybe "/:productId"

Reply:
Oh... I read your comment. So a single review. Can we make that explicit